src/Models/Argument/ArgumentDefinitions.cs

```diff
@@ -143,6 +143,60 @@ public static class Cosmos
         );
     }
 
+    public static class Postgres
+    {
+        public const string UserName = "user";
```

Comment:
Let's follow the name conventions

public const string UserName = "user-name";